### PR TITLE
add test_util to  make examples work well

### PR DESCRIPTION
**Pull request by jiangzhx**
# Which issue does this PR close?
add test_util to  make examples work well

### CODE REVIEW

File: ballista/core/src/lib.rs [hunk 1]
```diff
@@ -29,6 +29,7 @@ pub mod event_loop;
 pub mod execution_plans;
 /// some plugins
 pub mod plugin;
+pub mod test_util;
```
Comment (avantgardnerio) [inline ballista/core/src/lib.rs:32]: It seems questionable to export test dependencies as part of the main artifact. Perhaps a `#[cfg(test)]` would resolve that?

File: examples/Cargo.toml [hunk 2]
```diff
@@ -35,6 +35,7 @@ required-features = ["ballista/standalone"]
 
 [dependencies]
 ballista = { path = "../ballista/client", version = "0.10.0" }
+ballista-core = { path = "../ballista/core", version = "0.10.0" }
```
Comment (avantgardnerio) [inline examples/Cargo.toml:38]: Same... but `[test-dependencies]`... or maybe we need a `datafusion-test` package. I'm curious to hear what others think.

**Comment by jiangzhx** [comment]
Check Cargo.toml  got error;who can help to me to fix it.
_
Run # if you encounter error, try rerun the command below, finally run 'git diff' to
+ find . -mindepth 2 -name Cargo.toml -exec cargo tomlfmt -p '{}' ';'
 INFO 2023-02-06T04:21:24Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2023-02-06T04:21:24Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2023-02-06T04:21:24Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2023-02-06T04:21:24Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2023-02-06T04:21:24Z: cargo_tomlfmt: no problem found. good job! :)
 INFO 2023-02-06T04:21:24Z: cargo_tomlfmt: no problem found. good job! :)
 INFO [20](https://github.com/apache/arrow-ballista/actions/runs/4100201689/jobs/7071044470#step:8:21)23-02-06T04:[21](https://github.com/apache/arrow-ballista/actions/runs/4100201689/jobs/7071044470#step:8:22):24Z: cargo_tomlfmt: no problem found. good job! :)
warning: Not a git repository. Use --no-index to compare two paths outside a working tree
usage: git diff --no-index [<options>] <path> <path>

Diff output format options
_

File: ballista/core/src/lib.rs (hunk 1 above)
Comment (jiangzhx) [reply ballista/core/src/lib.rs]: thanks, i moved test_util to ballista-examples package.

**Pull request closed without merging**